src/mscorlib/corefx/System/Globalization/SortKey.cs

```diff
@@ -52,26 +52,6 @@ internal SortKey(String localeName, String str, CompareOptions options, byte[] k
             _options    = options;
             _string   = str;
         }
-
-        [OnSerializing]
```

Comment:
Anything on exposed types like this marked as [OnSerializing], [OnDeserializing], etc. should not be removed, as it's used via reflection as part of serialization/formatting.  If it was being removed before, it was a bug and should have been listed in the model.xml to prevent them from being removed.